walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -56,7 +56,7 @@
 
         // If there is no required attendee, return allMeetingTimes for 
         // optional attendees.
-        if (requiredAttendees.size() == 0) {
+        if (requiredAttendees.isEmpty()) {
```

Comment:
I just noticed that Line 53 can also be switched to use isEmpty().

Reply:
The condition is updated!